src/util/cookieUtils.js

```diff
@@ -0,0 +1,8 @@
+import querystring from 'qs';
+
+const isProd = process.env.NODE_ENV === 'production';
+export const MEMBER_COOKIE = isProd ? 'MEETUP_MEMBER' : 'MEETUP_MEMBER_DEV';
+
+export const parseMemberCookie = state =>
```

Comment:
Wondering if it makes sense to handle setting `id` to 0 and parseInt here, instead of a bunch of times throughout the code.

Also, then you can write some tests for this! (particularly I'm think about the cookie doesn't exist case, which will totally be a thing at some point soon).